metadata/metadata.go

```diff
@@ -91,15 +92,24 @@ func Pairs(kv ...string) MD {
 }
 
 // String implements the Stringer interface for pretty-printing a MD.
-// Ordering of the values is non-deterministic as it ranges over a map.
+// The value returned by this method is recommended only for diagnostic
+// purposes, and users are strongly discouraged from using this for anything
+// mission critical.
```

Comment:
I think it's worth stating somewhere that the ordering is guaranteed to be consistent.

Reply:
@easwars want me to? (since the original language came from you)

Reply:
How about this for the comment:

```go
// String implements fmt.Stringer to allow metadata to be printed when stored in a context.  It returns the metadata map as formatted by Go's fmt package.
```

@easwars ?

Reply:
Sounds good :)

Reply:
Done!